### ISSUE TITLE
Is this project being maintained 

### ISSUE THREAD

**Issue by ubaierbhat** [react-native-config/react-native-config#269]
@pedro what is the status of this project. Is it currently maintained or has the work on it stopped? I see the last commit was in January and there are many pending pull requests.

**Comment by sercanov** [comment]
Also compatibility issues with new RN versions

**Comment by SilencerWeb** [comment]
@erickreutz @jordanbrown

**Comment by drewandre** [comment]
Yeah, i've tried to get this to run on 0.56 and 0.54.4 and I've had no luck.

**Comment by drewandre** [comment]
I was able to run version 0.8.1 on react-native version 0.56. version 11.5 of react-native-config was returning an empty object when running on RN version 0.56.
